### PR TITLE
Bug 1583632, added a known issue

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1583632

@jeremyeder @mffiedler PTAL

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jeremyeder @mffiedler bump

**Comment by ahardin-rh** [comment]
@jeremyeder @mffiedler PTAL

File: install_config/registry/registry_known_issues.adoc [hunk 1]
```diff
@@ -16,6 +16,18 @@ toc::[]
...
+== Concurrent Build with Registry Pull-through
+
+The local *docker-registry* deployment takes on additional load. By default, it
+now caches content from *_registry.access.redhat.com_*. The images from
+*_registry.access.redhat.com_* for STI builds are now stored in the local
+registry. Attempts to pull them result in pulls from the local
+*docker-registry*. As a result, there are circumstances where extreme numbers of
+concurrent builds can result in timeouts for the pulls and the build can
+possibly fail. To alleviate the issue, scale the *docker-registry* deployment to
+more than one replica.
```
Comment (mffiedler) [inline install_config/registry/registry_known_issues.adoc]: Looks good.  The only addition might be a mention that the logs for the builder pod are where the timeouts can be seen.

**Comment by ahardin-rh** [comment]
@mffiedler Thanks! Updated.

**Review by mffiedler — APPROVED** [review]
/lgtm

**Pull request merged**